logging/logrus/middleware.go

```diff
@@ -21,12 +21,14 @@ var (
 //
 // All handlers will have a Logrus logger in their context, which can be fetched using `ctxlogrus.Extract`.
 func Middleware(entry *logrus.Entry, opts ...Option) httpwares.Middleware {
+	entry.Logger.Hooks.Add(&timeHook{})
```

Comment:
would prefer to just do this inline ... rather than adding `hooks`

Reply:
From the PR context and offline discussion, I can't think of a good way of doing this without hooks.